authors/apps/articles/exceptions.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.exceptions import APIException
+
+
+class NotFound(APIException):
```

Comment:
@myrdstom Good work here. Nice Initiative shown to leave the code base better than you found it.

Reply:
My bad